src/accordion/accordion-item.tsx

```diff
@@ -58,14 +66,13 @@ export const AccordionItem = ({
     // RENDER FUNCTIONS
     // =============================================================================
     // React spring animation configuration
-    const expandableStyles = useSpring({
-        height: expand ? resizeDetector.height : 0,
-    });
+    const resizeHeight = { height: expand ? resizeDetector.height : 0 };
+    const expandableStyles = useSpring(resizeHeight);
 
     const renderContent = () => {
         return (
             <Expandable
-                style={expandableStyles}
+                style={hasFirstLoad ? expandableStyles : resizeHeight}
```

Comment:
I think this is not necessary

Reply:
the animation affects how much we are able to scroll due to page height, and we need to scroll on page load